pages/landing/enterprise-eks/pricing/index.html

```diff
@@ -8,69 +8,143 @@
 permalink: /eks-for-enterprise/pricing/
 
 categories:
-  - title: Feature
+  - title: EKS
```

Comment:
Note, the hero subtext says:

> Sign up now for early access to Amazon EKS for Enterprise! Early access subscribers will receive exclusive benefits and a significantly reduced subscription price.

But the page doesn't make it clear what the early access benefits are? The previous version of the page had "standard" vs "early access" but now it's "standard" vs "enterprise," so it's less clear.

Reply:
Switched to be the same as heroku landing page in #582.